omf/blockmodel.py

```diff
@@ -24,7 +24,7 @@ class BaseBlockModel(ProjectElement):
         default="Z",
         length=1,
     )
-    corner = properties.Vector3(
+    origin = properties.Vector3(
```

Comment:
Super minor but do we also now want to update the comment to say "Origin ..." rather than "Corner ..." ?

Reply:
Also would it be possible to just define `origin `on the `ProjectElement `base class?

Reply:
That's where it lived in version 1 - but now we have `Composite` project-elements which don't have an origin.